README.md

```diff
@@ -321,6 +321,59 @@ travelTimeClient.timeFilter({
   .catch((e) => console.error(e));
 ```
 
+### Time Filter Many to Many Utility
+
+This utility function leverages `travelTimeClient.timeFilter` (as described in the previous section) to efficiently construct Many-to-Many matrices in batches.
+
+
+```ts
+const matrix = await travelTimeClient.manyToManyMatrix({
+    coordsFrom: [
+      { lat: 51.5055, lng: -0.0754 },
+      { lat: 51.5171, lng: -0.1062 },
+      { lat: 51.5309, lng: -0.1215 },
+      { lat: 51.5022, lng: -0.1149 },
+      { lat: 51.5144, lng: -0.1427 },
+    ],
+    coordsTo: [
+      { lat: 51.5055, lng: -0.0754 },
+      { lat: 51.5171, lng: -0.1062 },
+      { lat: 51.5309, lng: -0.1215 },
+      { lat: 51.5022, lng: -0.1149 },
+      { lat: 51.5144, lng: -0.1427 },
+    ],
+    transportation: { type: 'driving' },
+    properties: ['travel_time', 'distance'],
+    travelTime: 1800,
+    maxSearchesPerRequest: 2,
```

Comment:
I think we should remove `maxSearchesPerRequest` from readme for simplicity sake.